docker/gpu.Dockerfile

```diff
@@ -0,0 +1,95 @@
+# Stage1: Compile
+FROM nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04 AS compile-image
+# Install tvm
+ARG PYTHON_VER=3.7
+WORKDIR /root
+RUN apt-get update \
+ && apt-get install -y gcc \
+ libtinfo-dev \
+ zlib1g-dev \
+ build-essential \
+ cmake \
+ libedit-dev \
+ libxml2-dev \
+ libopenblas-dev \
+ ninja-build \
+ git \
+ wget \
+ llvm-10-dev \
+ python${PYTHON_VER} \
+ python${PYTHON_VER}-venv \
+ python${PYTHON_VER}-dev \
+ python3-pip \
+ && wget -q https://bootstrap.pypa.io/get-pip.py && python${PYTHON_VER} get-pip.py
+
+# Use venv
+ENV VIRTUAL_ENV=/venv
+RUN python3.7 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+COPY . /content
+# Build tvm
+WORKDIR /root
+RUN git clone https://github.com/apache/tvm tvm --recursive
+
+WORKDIR /root/tvm
+RUN mkdir -p build \
```

Comment:
Better to use
```sugguestion
RUN cd /root/tvm \
 && mkdir -p build \
```

Reply:
It's recommended to use WORKDIR refer to  https://github.com/hadolint/hadolint/wiki/DL3003